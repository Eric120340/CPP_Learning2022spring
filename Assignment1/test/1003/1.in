1 5
75 63 1 14 84 
5 6
25 8 100 4 35 68 
29 85 69 54 96 53 
93 18 100 15 96 72 
82 62 89 34 46 49 
70 1 76 32 1 41 
