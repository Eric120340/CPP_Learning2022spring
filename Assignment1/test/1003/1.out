10823 6925 19577 6881 9497 12641 
